### DISCUSSION TITLE
Remove builtin Position/Rotation/Matrix components from core library

### DISCUSSION THREAD

**Discussion by vkensou** [friflo/Friflo.Engine.ECS#26] [Ideas]
Many engine have there own vector/matrix struct. So embed Position/Rotation/Matrix/etc in core library is unnecessary. We can offer theres component as extensions.

**Comment by friflo** [comment]
Doing this will break compatibility.
So I don't want to do this at the moment.

Duplicate of https://github.com/friflo/Friflo.Engine.ECS/issues/14

**Reply by vkensou** [reply]
Since it's now a preview version, I don't think we need to worry too much about compatibility.

**Reply by friflo** [reply]
In case changing the mentioned types I would move them to namespace `Friflo.ECS.Math`.
I let this topic open for better naming suggestions.

**Comment by DruidOwl** [comment]
I have to place `global using Position = ...` in every project to resolve name ambiguines with `Friflo.Engine.ECS.Position` because I have my own `Position` struct fitting my needs.
